src/components/pages/Home/RenderHomePage.js

```diff
@@ -7,7 +7,7 @@ function RenderHomePage() {
   return (
     <div className="flux" style={{ textAlign: 'center' }}>
       <div className="user-info">
-        <h1>Hi {userData?.name}</h1>
+        <h1>Hi {userData.name}</h1>
```

Comment:
This change seems to be failing the tests.

I actually did the exact same thing myself thinking the `?` was a syntax error. It seems like the `?` is a sort of fallback which won't throw an error if the parameter `.name` does not exist.

Reply:
Thanks Harry!

Good catch. Fixing it and pushing.